src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        String [] data = new String[] {
+                "26.04.2019 John 4 50",
+                "05.04.2019 Andrew 3 200",
+                "10.04.2019 John 7 100",
+                "22.04.2019 Kate 9 100",
+                "25.06.2019 John 11 50",
+                "26.04.2019 Andrew 3 150",
+                "13.02.2019 John 7 100",
+                "26.04.2019 Kate 9 100"
+        };
+        String [] name = new String[] {
+                "John",
+                "Andrew",
+                "Kate"
+        };
+        String dateFrom = "01.04.2024";
```

Comment:
The variable `dateFrom` is assigned a future date which does not match the dates in the `data` array. Ensure that the date range is correct for the salary information you are trying to calculate.